### PR TITLE
nice logging, error handling

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

File: server/oracleService.js [hunk 1]
```diff
@@ -30,55 +31,63 @@ module.exports = class OracleService {
...
   };
 
   extendIfNeeded = async () => {
+    logger.debug("checking to extend oracle");
     const height = await this.aeternity.client.height();
 
     if (height > this.oracle.ttl - this.ttl / 5) {
       this.oracle = await this.oracle.extendOracle({type: 'delta', value: this.ttl});
-      console.log("Extended Oracle at height:", height, "new ttl:", this.oracle.ttl);
+      logger.debug("extended oracle at height:", height, "new ttl:", this.oracle.ttl);
```
Comment (kenodressel) [inline server/oracleService.js:50]: shouldnt this be a higher level than debug?

File: server/pageParser.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
 const SnippetLoader = require('./snippetLoader');
 const DomLoader = require("./domLoader");
 const Aeternity = require("./aeternity");
+let logger = require("./logger")(module);
 
 module.exports = class PageParser {
 
-  constructor(aeternity = null) {
+  constructor(aeternity = null, contextInfo) {
```
Comment (kenodressel) [inline server/pageParser.js:8]: required arguments should come before optional IMO

File: server/oracleService.js (hunk 1 above)
Comment (thepiwo) [reply server/oracleService.js:50]: It can be

File: server/pageParser.js (hunk 2 above)
Comment (thepiwo) [reply server/pageParser.js:8]: it's not required, I added a default null, I wouldn't like to put it before the aeternity client argument logically

**Pull request merged**